### DISCUSSION TITLE
I'm currently getting a captcha from connecting to serx.ml ?

### DISCUSSION THREAD

**Discussion by emmatebibyte** [searxng/searxng#1262] [Q&A] [answered]
I don't know where else to ask about this, but I'm currently getting captcha from https://serx.ml/

**Comment by return42** [comment]
Sorry, we are not the maintainer of https://serx.ml/ but this instance was added to searx.space in https://github.com/searxng/searx-instances/issues/64 .. may you like to contact @ice9smart

**Comment by ice9smart** [comment]
@emmatebibyte Hello! I'm the maintainer of _serx.ml_. If your IP sends more than 60 search requests in an hour, it will trigger a CAPTCHA to verify that you are not a bot. It uses Google reCAPTCHA v3, so that you only need to press on a button instead of figuring out which one is fire hydrant.

This function is powered by an open source nginx module [ngx_waf](https://github.com/ADD-SP/ngx_waf).

I used to use filtron, but it works poorly when using HTTP/3.

**Comment by return42** [comment, marked as answer]
>  It uses Google reCAPTCHA v3, 

Hm, thought we developed SearXNG to escape from google tracking ;-) .. why not using the limiter plugin of SearXNG?

> more than 60 search requests in an hour,

thats not much .. in the limiter we have sliding windows of 20sec (15 requests) and 10min (150 requests) ..

https://github.com/searxng/searxng/blob/e568da2314089a71666dfbccc8c6051c7fb7db78/searx/plugins/limiter.py#L50-L53

**Comment by emmatebibyte** [comment]
@ice9smart thanks for hosting the instance! This is a problem for me because I'm using librejs to filter proprietary js (and therefore if I want to pass the captcha every time I send more than 60 searches, I have to either disable the extension or create a temporary rule to allow recaptcha to load). I don't know if there's another solution for you in terms of captcha like a fake captcha honeypot for bots, but I appreciate the response.
